rcsbsearchapi/schema.py

```diff
@@ -112,9 +112,13 @@ def _make_group(fullname: str, nodeL: List) -> Union[SchemaGroup, Attr]:
     return group
 
 
-def _make_schema() -> SchemaGroup:
-    json1 = _load_json_schema()
-    json2 = _load_chem_schema()
+def _make_schema(structureSchemaURL: str, chemicalSchemaURL: str) -> SchemaGroup:
```

Comment:
I definitely like the cleanliness of passing in arguments here, but since `STRUCTURE_ATTRIBUTE_SCHEMA_URL` and `CHEMICAL_ATTRIBUTE_SCHEMA_URL` are constants that are already imported above, you probably don't need to do this.

Also just FYI for future reference, for this package, I believe variables should follow snake casing (e.g., it'd be `structure_schema_url` instead of `structureSchemaURL`)